symfony/routing/6.1/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "aliases": ["router"],
+    "conflict": {
+        "symfony/framework-bundle": "<5.3"
```

Comment:
6.1 isn't it, to get support for `type: attribute`

Reply:
It should be a conflict with `symfony/routing` isn't it? Or should I just keep `annotation` for now?

Reply:
my bad, nothing to do here, routing is already >= 6.1 by the name of the recipe and this is unrelated to fwb